core/src/main/java/dk/alexandra/fresco/logging/PerformanceLogger.java

```diff
@@ -17,19 +19,38 @@
   public DecimalFormat df = new DecimalFormat("#.00");
   
   public enum Flag {
-    LOG_NETWORK, LOG_RUNTIME, LOG_NATIVE_BATCH;
+    LOG_NETWORK, LOG_EVALUATOR, LOG_NATIVE_BATCH;
 
     public static final EnumSet<Flag> ALL_OPTS = EnumSet.allOf(Flag.class);
   }
 
   /**
-   * Prints any performance numbers picked up.
+   * Prints any performance numbers picked up using the
+   * default logger.
    */
-  public abstract void printPerformanceLog(int myId);
+  public default void printPerformanceLog(int myId){
+    log.info(makeLogString(myId));
+  }
```

Comment:
One request: The performance logger should not print it. Give this method a different object (PerformancePrinter) and create a reasonable default in the core test framework (which is the only use currently)

Reply:
I've made a suggestion implementation of a PerformancePrinter and refactored some code to it. Please check if it was something along those lines you had in mind.